packages/frontend/src/components/MarkdownRenderer/Components/NewsDrawerComponent.ts

```diff
@@ -0,0 +1,15 @@
+import MarkdownComponent from '.'
+
+export const NEWS_DRAWER_COMPONENT = MarkdownComponent({
+  styles: {
+    text: {
+      color: 'secondary.500',
+      textStyle: 'body-1',
+      fontSize: '1rem',
+    },
+    list: {
+      color: 'secondary.500',
+      marginInlineStart: '1.25em',
+    },
+  },
+})
```

Comment:
I moved it over to be a constant under `NewsItem` instead and I created `MarkdownRenderer` so that if we ever need markdown, we can just use this and customise however we want by changing the component